android/app/src/release/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+  package="host.exp.exponent"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <permission
```

Comment:
One quick question:  I see that this custom permission is included in both the `main` and `release` manifests:

```
<permission
    android:name="io.tidepool.urchin.permission.C2D_MESSAGE"
    android:protectionLevel="signature"/>
```

Does this need to be in both locations?

Reply:
I'm not certain that it needs to be in the release manifest. I'll test out removing it, and, if not needed, I'll create a separate PR for this fix